addons/mail/static/src/models/thread/thread.js

```diff
@@ -206,38 +206,34 @@ function factory(dependencies) {
             // relations
             if ('members' in data) {
                 if (!data.members) {
-                    data2.members = [['unlink-all']];
+                    data2.members = [unlinkAll()];
                 } else {
-                    data2.members = [
-                        ['insert-and-replace', data.members.map(memberData =>
-                            this.env.models['mail.partner'].convertData(memberData)
-                        )],
-                    ];
+                    data2.members = [insertAndReplace(data.members.map(memberData =>
```

Comment:
maybe just add a comment to be sure: we keep a list syntax because it is used in livechat override